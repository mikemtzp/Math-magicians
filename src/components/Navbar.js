// import React from 'react';

const Navbar = () => (
  <header>
    <h1><a href="/" className="logo">Math Magicians</a></h1>
    <ul>
      <li>
        <a href="/home">Home</a>
      </li>
      <li>
        <a href="/calculator">Calculator</a>
      </li>
      <li>
        <a href="/quote">Quote</a>
      </li>
    </ul>
  </header>
);

export default Navbar;
