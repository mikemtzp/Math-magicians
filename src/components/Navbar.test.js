import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter as Router } from 'react-router-dom';
import Navbar from './Navbar';

it('Renders Navbar component correctly', () => {
  const tree = renderer
    .create(
      <Router>
        <Navbar />
        {' '}
      </Router>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
