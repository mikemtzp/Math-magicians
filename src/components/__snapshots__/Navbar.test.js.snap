// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Navbar component correctly 1`] = `
Array [
  <nav
    className="navbar"
  >
    <h1>
      <a
        className="logo"
        href="/home"
        onClick={[Function]}
      >
        Math Magicians
      </a>
    </h1>
    <ul>
      <li>
        <a
          href="/home"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li>
        <a
          href="/calculator"
          onClick={[Function]}
        >
          Calculator
        </a>
      </li>
      <li>
        <a
          href="/quote"
          onClick={[Function]}
        >
          Quote
        </a>
      </li>
    </ul>
  </nav>,
  " ",
]
`;
